.github/workflows/cypress-altinn-app-frontend.yml

```diff
@@ -84,5 +87,5 @@ jobs:
       - name: Verify cypress and run tests
         run: |
           yarn run cy:verify
-          ./node_modules/.bin/start-server-and-test 'cd $GITHUB_WORKSPACE/src/altinn-app-frontend && yarn start' http://localhost:8080/altinn-app-frontend.js 'yarn run test:all:headless --env environment=tt02,testUserName=testuserexternal,testUserPwd=r@h74Rz7XYQJ'
+          ./node_modules/.bin/start-server-and-test 'cd $GITHUB_WORKSPACE/src/altinn-app-frontend && yarn start --no-hot' http://localhost:8080/altinn-app-frontend.js 'yarn run test:all:headless --config watchForFileChanges=false --env environment=tt02,testUserName=testuserexternal,testUserPwd=r@h74Rz7XYQJ'
```

Comment:
Are these changes required? If so, we should probably add them to the fork run as well: https://github.com/Altinn/app-frontend-react/blob/main/.github/workflows/cypress-altinn-app-frontend.yml#L87

Reply:
Not strictly necessary. I ran into the issue described here: https://github.com/cypress-io/github-action/issues/357 and I thought disabling hot reload would reduce listeners. Turned out I had to add the workaround to get it to work.

But I guess there is no need to have hot module reloading enabled in this context either way. Disabling in contributor run 👍

Reply:
Did now do much. Still makes little sense in enabling hot reload, so leaving it. Let me know if you feel otherwise.

Reply:
I agree, doesnt make sense to keep this enabled for testruns.